quickstart/src/main/java/com/microsoft/azure/kusto/quickstart/SampleApp.java

```diff
@@ -0,0 +1,589 @@
+package com.microsoft.azure.kusto.quickstart;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.kusto.data.Client;
+import com.microsoft.azure.kusto.data.ClientFactory;
+import com.microsoft.azure.kusto.ingest.IngestClient;
+import com.microsoft.azure.kusto.ingest.IngestClientFactory;
+import com.microsoft.azure.kusto.ingest.IngestionMapping;
+import com.microsoft.azure.kusto.ingest.IngestionProperties;
+import org.apache.commons.lang3.StringUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Scanner;
+import java.util.UUID;
+
+/**
+ * SourceType - represents the type of files used for ingestion
+ */
+enum SourceType {
+    localFileSource, blobSource
```

Comment:
java enum members are usually in SHOUTY_CASE

Reply:
Wrote it like that to automatically identify it from JSON. should i look for ways to convert like we did in python?

Reply:
Does giving them string values don't work? (Like you did in the next enum)